### ISSUE TITLE
Final assertion in AboutAssignmentAndArithmetic needs more explanation

### ISSUE THREAD

**Issue by theosinitsin** [vexx32/PSKoans#409]
# Describe "Koan Bug, Issue, or Help Request"

AboutAssignmentAndArithmetic

## Context "The Problematic Assertions"

```
    It 'can get a bit confusing to follow' {
        $Value = __

        $Value /= 3
        $Value %= 5
        $Value += 4
        $Value *= 7
        $Value -= 7

        $Value | Should -Be 42
    }
}

```

## Context "Your Attempts"

Tried many things, including working in reverse from the bottom up (starting with 42 as the result).

## Context "Additional Information"

I think this assertion needs some additional explanation or hints. I was only able to get the right answer by guessing and incrementing by one, until I found a number that worked.

**Comment by vexx32** [comment]
Yeah, this one's... Hmm. Most of it's straightforward, but the fact that modulo is included means that there are many potentially correct answers.

42 - 7 -> 35
35 / 7 -> 5
5 - 4 -> 1

Because the modulo is there, anything that gives a remainder of 1 when divided by 5 would work for that step - so you could work from 6, 11, 16, 21, 26, 31, 36 (etc.)

And the first division by 3 just means you multiply any of those by 3 to get a valid starting point. Probably better to find a more suitable sequence of operations for this one, I reckon. Modulus is a bit hard to reverse-engineer in the same way.
